addon/page-object.js

```diff
@@ -87,6 +84,10 @@ export default class PageObject {
     });
   }
 
+  clickWithText(rawSelector, text) {
```

Comment:
Dropping the preposition ("with", and yes, I had to look up the part of speech it was) `clickText()` might also be appropriate here, as you already have `clickButton()` which does not have a preposition (`clickOnButton` would be with a preposition).